package.json

```diff
@@ -29,7 +29,7 @@
   },
   "homepage": "https://github.com/Eppo-exp/node-server-sdk#readme",
   "dependencies": {
-    "@eppo/js-client-sdk-common": "4.0.1",
+    "@eppo/js-client-sdk-common": "4.0.2",
```

Comment:
Should this perhaps be `4.0.*`?  I would assumed we'd want to pick up patches automatically as they should be non-breaking.

Reply:
We don't want to do that just so that we don't actually break things for customers who did not upgrade by pushing a bad common package (which is hopefully something that doesn't happen)